src/mon/MonClient.cc

```diff
@@ -424,6 +424,11 @@ void MonClient::handle_monmap(MMonMap *m)
     }
   }
 
+  // Update cct with MON addresses
+  std::list<entity_addrvec_t> mon_addrs;
```

Comment:
As fetching the mon addrs from cct is in MonMap, moving the  store of the same into MonMap, makes for a better abstraction.